README.md

```diff
@@ -68,6 +68,41 @@ bandcamp.search(params, function(error, searchResults) {
 [View example with output](examples/search.js).
 
 
+### `tag(params, callback)`
```

Comment:
Probably should have a better name for this api. `getTaggedAlbums` or `getAlbumsWithTags` perhaps?